app/src/main/java/com/thanhclub/musicdemo/services/PlaySongService.java

```diff
@@ -0,0 +1,149 @@
+package com.thanhclub.musicdemo.services;
+
+import android.app.Notification;
+import android.app.PendingIntent;
+import android.app.Service;
+import android.content.Intent;
+import android.os.Binder;
+import android.os.Build;
+import android.os.IBinder;
+import android.support.annotation.Nullable;
+import android.util.Log;
+import android.widget.RemoteViews;
+
+import com.thanhclub.musicdemo.R;
+import com.thanhclub.musicdemo.manager.SongPlayerManager;
+import com.thanhclub.musicdemo.model.Song;
+import com.thanhclub.musicdemo.ui.MainActivity;
+
+import java.util.ArrayList;
+
+public class PlaySongService extends Service {
+    private SongPlayerManager playerManager;
+    private ArrayList<Song> songs;
```

Comment:
em nên khai báo list thay vì ArrayList
https://stackoverflow.com/questions/2279030/type-list-vs-type-arraylist-in-java